backend/doc/setting.md

```diff
@@ -436,4 +436,30 @@ to have more/less log about something.
 
 You can change default color of apps by setting color.json file, by default,
 placed at root of tracim dir, see [color.json.sample](../../color.json.sample)
-for default config file.
\ No newline at end of file
+for default config file.
+
+# Search method using elastic_search (tracim 2.3+) #
+
+First, you need an elastic_search server, and easy way to have this is with docker:
+
+`docker run -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" -e "cluster.routing.allocation.disk.threshold_enabled=false" elasticsearch:7.0.0`
+
+You then need to setup config file:
+
+    search.enabled = True
+    search.elasticsearch.host = localhost
+    search.elasticsearch.port = 9200
+
+You're elasticsearch server need to be running. You can then setup index with:
+
+`tracimcli search init`
+
+You can resync data with:
+
+`tracimcli search add`
+
+If there is an update of tracim, use this one to migrate index:
+
+`tracimcli search update`
```

Comment:
i agree.

Reply:
Do you have changed it ? If yew, you can resolve this conversation.

Reply:
@inkhey @buxx what does update/upgrade do ? reindex all the contents? detect deprecated data and index it? its not clear what it does.

Reply:
(@inkhey corerct me if i am wrong) upgrade is: create new index, index all content was indexed in "old" index. replace old index by new index.

Action needed when indexing "rules" have been modified, like in new tracim release.

Reply:
so it's not an upgrade but a reindex, doesn't it?

Reply:
@lebouquetin as the mecanism create a new index, it's different to just reindex,
@buxx yes.

Reply:
@inkhey as discussed, mark/document the `upgrade` command as *experimental*.
Also, document the way to rebuild the index in case it is corrupted. (should we do delete/index ? delete/init/index ?)

Reply:
done